test/fixtures/assign_users.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```

Comment:
With the project scope, we don't need the test case, so pls don't commit test cases file.